packages/lambdas/src/sqs-to-fhir.ts

```diff
@@ -128,21 +117,7 @@ export async function handler(event: SQSEvent) {
       };
 
       log(`Converting payload to JSON, length ${payloadRaw.length}`);
-      let payload: any; // eslint-disable-line @typescript-eslint/no-explicit-any
-      if (isSandbox()) {
-        const idsReplaced = replaceIds(payloadRaw);
-        log(`IDs replaced, length: ${idsReplaced.length}`);
-        const placeholderUpdated = idsReplaced.replace(placeholderReplaceRegex, patientId);
-        payload = JSON.parse(placeholderUpdated);
-        log(`Payload to FHIR (length ${placeholderUpdated.length}): ${JSON.stringify(payload)}`);
-      } else {
-        payload = JSON.parse(payloadRaw);
-      }
-
-      // light validation to make sure it's a bundle
-      if (payload.resourceType !== "Bundle") {
-        throw new Error(`Not a FHIR Bundle`);
-      }
+      const payload: Bundle = parseRawBundleForFhirServer(payloadRaw, patientId, log);
```

Comment:
can we be explicit here in PR description and also what bugs are being solved, since this change is making it so that we replaceIds for all bundles, whether in sandbox or in prod. 

Afail we had a discussion hypothesizing how due to duplicates across xml files + deterministic uuid generation, we were getting HAPI insert errors because two resources with the same id were getting inserted simaltaneously from different bundles

Reply:
we also hypothesized that the reason we observed a difference in the number of resources in the FHIR server vs after conversion in the e2e fhir tests was because of this (hapi deduplicating due to identical ids)

Reply:
> this change is making it so that we replaceIds for all bundles, whether in sandbox or in prod

No, the code removed on the left was moved to the `parseRawBundleForFhirServer` function.

AFAIK we're not introducing any change in logic related to replacing IDs, just moved the code to a function so it can be called after we load the bundle.

Reply:
sorry my bad you are right